### PR TITLE
rasterize chromium image

### PR DESCRIPTION
**Pull request by orenzohar**

### CODE REVIEW

File: docker/rasterize-chrome/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+&& pip install --no-cache-dir -r requirements.txt \
+&& apt-get purge -y --auto-remove \
+  gcc \
+  python-dev \
+&& rm -rf /var/lib/apt/lists/*
+
+# Install Chrome
+RUN apt-get update
+RUN wget https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb
+RUN dpkg -i google-chrome-stable_current_amd64.deb; apt-get -fy install
```
Comment (glicht) [inline docker/rasterize-chrome/Dockerfile]: Combine the above three commands into one and add cleanup. Also, not sure about the apt-get install part and if it is needed.

**Comment by orenzohar** [comment]
@glicht seems like `wget` fails in getting the latest chrome from google (locally too, looks like a ban for too many requests). I will upload the zip into the docker directory for now

File: docker/rasterize-chrome/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  python-dev \
+  wget \
+  libxss1 \
+  libappindicator1 \
+  libindicator7 \
+&& pip3 install --no-cache-dir -r requirements.txt
+
+# Install Chrome
+#RUN wget https://dl.google.co/linux/direct/google-chrome-stable_current_amd64.deb; \
+RUN dpkg -i google-chrome*.deb; \
```
Comment (glicht) [inline docker/rasterize-chrome/Dockerfile]: Use `&&` and not `;`

Comment (orenzohar) [reply docker/rasterize-chrome/Dockerfile]: && will not run the next command on an exit code other than 0

File: docker/chromium/Dockerfile [hunk 3]
```diff
@@ -1,8 +1,26 @@
...
+  python-dev \
+  wget \
+  libxss1 \
+  libappindicator1 \
+  libindicator7 \
+&& pip3 install --no-cache-dir -r requirements.txt
+
+# Install Chrome
+#RUN wget https://dl.google.co/linux/direct/google-chrome-stable_current_amd64.deb; \
+RUN dpkg -i google-chrome*.deb; \
```
Comment (glicht) [inline docker/chromium/Dockerfile]: Use `&&` instead of `;`

File: .dependabot/config.yml [hunk 4]
```diff
@@ -502,3 +502,14 @@ update_configs:
         dependency_name: demisto/*
         update_type: all
 
+  - package_manager: python
```
Comment (glicht) [inline .dependabot/config.yml]: Not needed as you've changed the name of the project to am existing one.

File: docker/chromium/Dockerfile (hunk 3 above)
Comment (orenzohar) [reply docker/chromium/Dockerfile]: > && will not run the next command on an exit code other than 0 (which is expected in this scenario)
@glicht

Comment (glicht) [reply docker/chromium/Dockerfile]: ok. so use && for the next steps. and also add a verification that the install actually installed chrome. You can run something of the form: `/opt/google/chrome/chrome --version | grep 76.0.3809`

File: docker/chromium/Dockerfile [hunk 5]
```diff
@@ -1,8 +1,26 @@
...
+apt --fix-broken -y install; \
+apt-get -y install unzip
 
-RUN apk --update add --no-cache --virtual .build-dependencies python-dev build-base wget git \
-  && pip install --no-cache-dir -r requirements.txt \
-  && apk del .build-dependencies
+# Clean Up
```
Comment (glicht) [inline docker/chromium/Dockerfile]: Don't do the cleanup in a separate RUN. It should be done within each separate RUN.

File: docker/chromium/Dockerfile (hunk 3 above)
Comment (orenzohar) [reply docker/chromium/Dockerfile]: > `&& python3 -c "from webdriver_manager.chrome import ChromeDriverManager; ChromeDriverManager().install()" \`

I added a python command that will ensure that chrome installed successfully, and will init the web driver so the

**Comment by orenzohar** [comment]
@glicht updated the PR for the review

File: docker/chromium/Dockerfile (hunk 5 above)
Comment (orenzohar) [reply docker/chromium/Dockerfile]: if I do the clean up before the dpkg, the installation will fail

File: docker/chromium/Dockerfile (hunk 3 above)
Comment (glicht) [reply docker/chromium/Dockerfile]: Excellent.

File: docker/chromium/Dockerfile (hunk 5 above)
Comment (glicht) [reply docker/chromium/Dockerfile]: I added a commit regarding what I meant about cleanup. You can see that now the image on docker hub is smaller.

**Pull request merged**